README.Rmd

```diff
@@ -83,33 +83,7 @@ The repository template includes the following:
-1. Assuming you work under agile/scrum, then create a product backlog, prioritize and make a sprint plan.
-2. The intention is always to re-use as much as possible from `{admiral}` core package. If you find anything additional needed for the package extension, you should first question whether it might be a common need for other TAs and if so consider instead raising an issue to `{admiral}` core. When designing new functions always try to stay aligned with the `r dyn_link("programming strategy.", admiraldev_homepage, "articles/programming_strategy.html")`. 
-3. Start development of your foundational first release 0.1.0. Follow a consistent `r dyn_link("development process", admiraldev_homepage, "articles/development_process.html")` to `{admiral}`. 
-4. Line up testers from your companies and others and set expectations around when you believe a stable version would be available for user testing. You can use the admiral Slack community to raise interest to get involved.
-5. Add a pharmaverse badge to your README: https://pharmaverse.org/contribute/badges/ - needs support from a pharmaverse council rep. 
-6. Raise an `{admiral}` repo issue to ensure your package extension site is linked from the core `{admiral}` site `r dyn_link("here.", admiral_homepage, "index.html#types-of-packages")`
-7. It is important that the `{admiral}` family of packages keep to a similar release schedule and cadence, in order to ease adoption by our users and to give clear expectations. The `{admiral}` core package cadence of releases is one every quarter on a fixed schedule (every first Monday of the last month of a quarter - March, June, September, December). The core package will set the release schedule for the package extensions to follow, i.e. once `{admiral}` releases we'd expect package extension releases targeted within a 2 week window. These releases are communicated via our Slack channel as well as at our quarterly user community meetings.
-8. Once you are happy your package extension has been well tested and is at a sufficient state then make a submission to CRAN. The technical lead should be named as maintainer. After the CRAN release, you should advertise this via Slack & LinkedIn.
-9. Plan any future further enhancements and make issues. When your team feels ready you can open up to development contributions for these from the wider community - see `r dyn_link("this page.", admiral_homepage, "CONTRIBUTING.html#type-1-contribution-with-code")`. Please use the _“good first issue”_ (ideal for new starters) & _“help wanted”_ (ideal for more experienced contributors) issue labels.
+For detailed information regarding the ethos of `{admiral}` package extensions and how to decide if you should set your own one up, please see our [Package Extension Guidance](https://pharmaverse.github.io/admiraldev/main/articles/package_extensions.html). You will also find details about team setup, suggested workflow and lessons learned from past package extensions.
```

Comment:
see comment above

Reply:
With the adoption of GitHub Flow I believe we can drop the `main` `devel` stuff and just link to the generic site.